manifests/tftp.pp

```diff
@@ -29,4 +29,91 @@
       root => $foreman_proxy::tftp_root,
     }
   }
+
+  case $::osfamily {
+    'RedHat': {
+      if versioncmp($::operatingsystemmajrelease, '6') <= 0 {
```

Comment:
Please use something like https://github.com/theforeman/puppet-puppet/blob/master/manifests/params.pp#L319 here, as $::operatingsystemmajrelease is (depending on the facter version) not a builtin fact.